.github/workflows/analyse-pr.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           ref: ${{ github.event.pull_request.head.sha }}
           fetch-depth: 0
       - name: Set up JDK 11
-        uses: actions/setup-java@v3
+        uses: actions/setup-java@v2
```

Comment:
is that one needed? or are you just debugging 😋

Reply:
Just shooting in the air a little bit, reverting the last commits